### PR TITLE
updated readme documents

### PR DESCRIPTION
**Pull request by zacharyrobin**
Updates to some of our readme documents.

### CODE REVIEW

File: EndToEndSetupGuide.md [hunk 1]
```diff
@@ -62,157 +81,137 @@ git clone https://github.com/mcode/rems-setup.git
...
+	# Navigate to the frontend directory
+	cd frontend
+	
+	# Setup
+	npm install
 
-	`npm start`
+	# Run
+	npm start # Mac or Linux
+	PORT=5050 npm start # Windows
```
Comment (smalho01) [inline EndToEndSetupGuide.md]: do we need port here?

File: EndToEndSetupGuide.md [hunk 2]
```diff
@@ -62,157 +81,137 @@ git clone https://github.com/mcode/rems-setup.git
...
-
-		`npm start`
-	- Windows
-
-		`PORT=4040 npm run start`
-	
-	
+	# Run
+	npm start # Mac or Linux
+	PORT=4040 npm run start # Windows
```
Comment (smalho01) [inline EndToEndSetupGuide.md]: do we need port here for windows?

**Review by smalho01 — CHANGES_REQUESTED** [review]

File: EndToEndSetupGuide.md (hunk 1 above)
Comment (plarocque4) [reply EndToEndSetupGuide.md]: yes, without the port it'll run at the default port which is incorrect.

File: EndToEndSetupGuide.md (hunk 2 above)
Comment (plarocque4) [reply EndToEndSetupGuide.md]: Yes, these are on purpose.

Comment (plarocque4) [reply EndToEndSetupGuide.md]: The port was only needed on windows, but you could just specify it for all. This seems weird to put the comment # Windows on the same line, seems wrong to me.

File: EndToEndSetupGuide.md (hunk 1 above)
Comment (plarocque4) [reply EndToEndSetupGuide.md]: This is like the other one, should be fine to specify the port for all platforms I believe.

Comment (plarocque4) [reply EndToEndSetupGuide.md]: Having a comment on the line with the code seems like a bad idea when someone is trying to copy and paste it in a terminal.

File: EndToEndSetupGuide.md (hunk 2 above)
Comment (plarocque4) [reply EndToEndSetupGuide.md]: Having a comment on the line with the code seems like a bad idea when someone is trying to copy and paste it in a terminal.

**Review by smalho01 — APPROVED** [review]

**Pull request merged**